i18n/locale_fr.xml

```diff
@@ -78,7 +78,7 @@
         <text name="guidanceSteering_setting_header_headland" text="Gestion des fourrières"/>
         <text name="guidanceSteering_setting_scopeTrackFarmId" text="Sauvegarder la trace pour la ferme."/>
         <text name="guidanceSteering_tooltip_scopeTrackFarmId" text="Sauvegarde de la piste limitée à la ferme du véhicule, par défaut elle est accessible à tous."/>
-        <text name="guidanceSteering_tooltip_offsetLines" text="Décalez la ligne pour augmenter la garde au sol."/>
-        <text name="guidanceSteering_setting_offsetLines" text="Décalage de la ligne"/>
+        <text name="guidanceSteering_tooltip_offsetLines" text="Modifier la hauteur des lignes pour améliorer leur visibilité dans des cultures hautes."/>
```

Comment:
Thanks to @KITT3000 who explained me that this is for height of the lines. I submit a proper FR translation.